src/js_results.hpp

```diff
@@ -160,17 +160,22 @@ inline void setup_aliases(parser::KeyPathMapping &mapping, const realm::SharedRe
 {
     const realm::Schema &schema = realm->schema();
     for (auto it = schema.begin(); it != schema.end(); ++it) {
+        TableRef table;
```

Comment:
Init to null.

Reply:
I don't think it is needed since `TableRef`'s default constructor is setting it to `null`.